### PR TITLE
perf README.md & Support problem package import

### PR DESCRIPTION
**Pull request by Zztrans**
We used `mirage_db` to init PG data and `test` to init minio data before, which seem to be not consistent.

Currently, all use `go test`.

Also perf the README.md.

### CODE REVIEW

File: tests/core/pg_test.go [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+	mapper.CreateUser(db, model.User{
+		Name:     "anonymous",
+		Account:  "anonymous",
+		Password: func() *string { s := "anonymous"; return &s }(),
+		Roles: []*model.Role{
+			{Name: "anonymous"},
+		},
+	})
+}
```
Comment (KanadeSiina) [inline tests/core/pg_test.go:53]: Maybe we should make the test idempotent, and use an additional script to import problem package.

**Review by KanadeSiina — CHANGES_REQUESTED** [review]
See my comment, an additional script is preferred.

**Comment by Zztrans** [comment]
> See my comment, an additional script is preferred.

script `read_pkg` has been added.

File: tests/core/minio_test.go [hunk 2]
```diff
@@ -1,55 +1,54 @@
...
+	// 	_, minioErr := minioClient.FPutObject(ctx, bucketName,
+	// 		relativePath,
+	// 		path,
+	// 		minio.PutObjectOptions{})
+	// 	if minioErr != nil {
+	// 		log.Fatalln(minioErr)
+	// 	}
+	// 	return minioErr
+	// })
 }
```
Comment (slhmy) [inline tests/core/minio_test.go:54]: Simply remove these code will be find

File: tests/mapper/user_test.go [hunk 3]
```diff
@@ -15,7 +15,7 @@ func TestUserMapper(t *testing.T) {
 	user := model.User{
 		Account:  "test",
 		Password: func() *string { s := "test"; return &s }(),
-		Roles:    []*model.Role{{Name: "admin"}},
+		// Roles:    []*model.Role{{Name: ""}}, // If added, foreign key constraint make it can't be delete.
```
Comment (slhmy) [inline tests/mapper/user_test.go:18]: We should fix this, actually I remember there's something similiar with this problem in `problem` mapper.

File: src/application/init_db/main.go [hunk 4]
```diff
@@ -14,26 +14,17 @@ func main() {
...
-	})
+	// description := `Write a program that prints "Hello World!".`
+	// mapper.CreateProblem(db, model.Problem{
+	// 	Slug:        "hello-world",
+	// 	Title:       "Hello World!",
+	// 	Description: &description,
+	// 	Tags: []*model.AlgorithmTag{
+	// 		{Name: "Primer"},
+	// 	},
+	// })
```
Comment (slhmy) [inline src/application/init_db/main.go:27]: Remove these code will be fine.

**Review by slhmy — CHANGES_REQUESTED** [review]
Almost good enough ~

**Review by slhmy — APPROVED** [review]

**Comment by slhmy** [comment]
Still has something need to perf (like we don't leave dead code too often)

But I don't want to hang too long on this PR, so merging it.

**Pull request merged**